doc: update configuration tool description of vuarts (#5536)
projectacrn/acrn-hypervisor: doc/tutorials/acrn_configuration_tool.rst

``SERIAL_CONSOLE`` (a child node of ``DEBUG_OPTIONS``):
  Specify the host serial device is used for hypervisor debugging.

>   This configuration is valid only if SOS legacy_vuart0 is enabled.

Reviewer: ```suggestion
  This configuration is valid only if Service VM ``legacy_vuart0`` is enabled.
```

Please use "Service VM" instead of "SOS" throughout the document.

Author: updated.

  Leave this field empty if SOS console_vuart is enabled. Using

>   bootargs for console_vuart configuration.

Reviewer: ```suggestion
  ``bootargs`` for ``console_vuart`` configuration.
```

Author: updated.

  must exactly match the module tag in the GRUB multiboot cmdline.

``bootargs`` (a child node of ``os_config``):

>   Specify the kernel boot arguments in bootargs under the parent of board_private.

Reviewer: ```suggestion
  Specify the kernel boot arguments in ``bootargs`` under the parent of ``board_private``.
```

Parent is `board_private` ou should it be `os_config`?

Author: updated.

Should be ``board_private``. There are several``bootargs`` in place. However, for the service os, the ``bootargs`` under ``os_config`` should is not configurable (configurable="0") and default value is "SOS_VM_BOOTARGS" which indicates it refers to the ``bootargs`` under ``board_private``.

Reviewer: So is the definition given on line 298 correct?

Author: I edited a wrong one, just add `` as tag. Thanks.

  Refer to :ref:`vuart_config` for detailed vUART settings.

``console_vuart``:
  Specify the console vUART (aka PCI based vUART) with the vUART ID by
  its ``id`` attribute.
  Refer to :ref:`vuart_config` for detailed vUART settings.

``communication_vuart``:
  Specify the communication vUART (aka PCI based vUART) with the vUART ID by
  its ``id`` attribute.
  Refer to :ref:`vuart_config` for detailed vUART settings.

``type`` (a child node of ``legacy_vuart``):
  vUART (aka COM) type; currently only supports the legacy PIO mode.

``base`` (a child node of ``legacy_vuart``,``console_vuart``,``communication_vuart``):
  vUART (A.K.A COM) enabling switch. Enable by exposing its COM_BASE
  (SOS_COM_BASE for Service VM); disable by returning INVALID_COM_BASE.

  console and communication vUART (A.K.A PCI based vUART) enabling switch.
  Enable by specifying PCI_VUART; disable by returning INVALID_PCI_BASE.

``irq`` (a child node of ``legacy_vuart``):
  vCOM IRQ.

``target_vm_id`` (a child node of ``legacy_vuart1``, ``communication_vuart``):
  COM2 is used for VM communications. When it is enabled, specify which
  target VM the current VM connects to.

>   communication_vuart is used for VM communications. When it is enabled, specify

Reviewer: ```suggestion
  ``communication_vuart`` is used for VM communications. When it is enabled, specify
```

Author: updated

  which target VM the current VM connects to.

``target_uart_id`` (a child node of ``legacy_vuart1``,``communication_vuart``):
  Target vUART ID to which the vCOM2 connects.

>   Target vUART ID to which the communication_vuart connects.

Reviewer: ```suggestion
  Target vUART ID to which the ``communication_vuart`` connects.
```

Author: updated
